dwave/system/samplers/clique.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import math
+
+import dimod
+import dwave_networkx as dnx
+
+from minorminer.busclique import find_clique_embedding, busgraph_cache
+
+from dwave.system.samplers.dwave_sampler import DWaveSampler
+
+__all__ = ['DWaveCliqueSampler']
+
+
+class DWaveCliqueSampler(dimod.Sampler):
+    """A Sampler for solving clique problems on the D-Wave system.
+
+    The `DWaveCliqueSampler` wraps
+    :func:`minorminer.busclique.find_clique_embedding` to generate embeddings
+    with even chain length. These embeddings will work well for dense
+    binary quadratic models. For sparse models, using
+    :class:`.EmbeddingComposite` is preferred.
+
```

Comment:
Can you add a reference to https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/samplers.html#dwavesampler so new users see what a DWaveSampler is.